laborator/content/etapele-compilarii/solution/05-lib-fix/add.c

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+int add(int a, int b)
+{
+	return a+b;
```

Comment:
```suggestion
	return a + b;
```
Same for the skelton.